ReactWindows/ReactNative/Views/Text/ReactTextShadowNode.cs

```diff
@@ -198,6 +250,127 @@ public void SetFontStyle(string fontStyleString)
         }
 
         /// <summary>
+        /// Sets the background color for the node.
+        /// </summary>
+        /// <param name="backgroundColor">The masked color value.</param>
+        [ReactProp(ViewProps.BackgroundColor, CustomType = "Color")]
+        public void SetBackgroundColor(uint? backgroundColor)
+        {
+            if (_isVirtual)
+            {
+                ThrowException("backgroundColor");
```

Comment:
Can you rename this method to 'ThrowNotSupportedException' to make it more explicit what's being thrown?